src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,76 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+class BackupConfiguration(
+    private val driver: WebDriver,
+    private val access: AdminAccess
+) {
+    private val logger = LogManager.getLogger(this::class.java)
+    private val deleteBackupLocator = By.xpath(
+        "//*[@id='tbl_services']" +
+            "//*[contains(text(), 'com.atlassian.jira.service.services.export.ExportService')]" +
+            "/ancestor::tr" +
+            "//*[contains(@class, 'operations-list')]" +
+            "//*[contains(text(), 'Delete')]"
+    )
+
+    /**
+     * By default, this function will try to delete 10 backup services.
+     * If your instance has less than 10 backup services it will remove all of them.
+     * If your instance has more than 10 backup services it will remove only 10.
+     * Recommended function to use in case of more than 10 backups is {@link #disable(Int)}
+     * where you can explicitly set a number of services to delete.
+     */
+    fun disable(): BackupConfiguration {
```

Comment:
Let's call it `delete`.

Reply:
Here, I followed the pattern in `RichTextEditorConfiguration`, but in this case, the actual action is just `delete`. I will rename it.

Reply:
Well, the RTEC actually does *disable* RTE and it doesn't *delete* RTE. It's reversible, not destructive.

Reply:
PS. RTEC describes an actual page - there is such a thing as "Rich Text Editor configuration" page. There is no "backup configuration" page. But we can leave that in and pretend the "Services" page acts like a backup config page. We can always add the actual services page in the future.